packages/eas-build-job/src/common.ts

```diff
@@ -115,7 +115,10 @@ export interface Cache {
   disabled: boolean;
   clear: boolean;
   key?: string;
-  cacheDefaultPaths: boolean;
+  /**
+   * @deprecated We don't cache anything by default anymore.
+   */
+  cacheDefaultPaths?: boolean;
```

Comment:
Remove this from the joi schema below (line 125).

Reply:
Actually, shouldn't we remove the field altogether?

Reply:
that would be a breaking change, shouldn't we warn users first?

Reply:
I see, so actually we should do the opposite - leave the joi schema as is and we can remove the `cacheDefaultPaths` field from type?

Reply:
But can we distinguish if the `cache` object was set explicitly or if we got default values? FWIW `cacheDefaultPaths` defaults to true.

Reply:
default is applied by eas-build-job schema, when we read eas.json in cli, there is not default yet

Reply:
>But can we distinguish if the cache object was set explicitly or if we got default values? FWIW cacheDefaultPaths defaults to true.
Yes, we can do this in EAS CLI. See https://github.com/expo/eas-cli/pull/1768